### DISCUSSION TITLE
React components as satori arguments?

### DISCUSSION THREAD

**Discussion by kbkn3** [vercel/satori#439] [Q&A]
Is there any way to pass react components as satori arguments?

like

```
const Component = React.memo((props)=>{
return(
  <p>{props}<p/>
)});
```

```
const svg = await satori(Component,...)
```

I am struggling with this because I want to pass arguments to the component to change it flexibly.

**Comment by shuding** [comment]
You can only use pure JSX or functions without any React API, because Satori itself doesn't support React but only JSX.
